src/Apps/CollectorProfile/collectorProfileRoutes.tsx

```diff
@@ -113,4 +166,82 @@ export const collectorProfileRoutes: AppRouteConfig[] = [
       },
     ],
   },
+  {
+    path: "/collector-profile/my-collection/artworks/new",
```

Comment:
**Suggestion:** maybe we could add these routes as children to `my-collection` on line 123?

Reply:
I tried, it didn't work for me, got 404 error